lib/govuk_docker_cli.rb

```diff
@@ -53,4 +53,10 @@ def prune
   def run(*args)
     Commands::Run.new(options[:stack], args).call
   end
+
+  desc "startup [APP_STACK]", "Run the service in the current directory with the `app` stack. Variations can be provided, for example `live` or `draft`."
+  def startup(app_stack = nil)
+    stack = app_stack ? "app-#{app_stack}" : "app"
```

Comment:
Minor, but could we use 'variation' instead of 'app_stack', so we don't have too many stack concepts floating around?

Reply:
Agreed, I've amended that now.